openedx_external_enrollments/external_enrollments/greenfig_external_enrollment.py

```diff
@@ -98,3 +100,45 @@ def _get_course_list(self, course_settings):
             return str(error), status.HTTP_500_INTERNAL_SERVER_ERROR
         else:
             return response
+
+    def _post_enrollment(self, data, course_settings=None):
```

Comment:
We are duplicating code. This is the same method defined in the base class. @Jacatove